index.html

```diff
@@ -10,346 +10,305 @@
     <meta name="description" content="mobile first web design" />
     <link rel="icon" type="image/png" sizes="32x32" href="images/logo.png" />
     <link rel="stylesheet" href="css/styles.css" />
-    <link
-      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
-      rel="stylesheet"
-    />
-    <script
-      src="https://kit.fontawesome.com/6d267a36ba.js"
-      crossorigin="anonymous"
-    ></script>
-    <style>
-      li {
-        display: inline;
-        padding: 5px;
-      }
-    </style>
   </head>
   <body class="main">
     <!-- navbar setup -->
     <header>
+      <!-- phone navigation -->
+      <section class="phone-navigation">
+        <a href="/" class="logo">
+          <h3>Welcome</h3>
+        </a>
+        <button type="button" class="menu-button">
+          <img src="../assets/Union.svg" alt="menu link" />
+        </button>
+        <nav class="mobile-navigation hide">
+          <button type="button" class="close-button">
+            <img src="../assets/Icon-Cancel.svg" alt="cancel icon" />
+          </button>
+          <ul>
+            <li>
+              <a href="#" class="port-link"> Portfolio </a>
+            </li>
+            <li>
+              <a href="#" class="abt-link"> </a>
+            </li>
+            <li>
+              <a href="#" class="contact-link"> Contact </a>
+            </li>
+          </ul>
+        </nav>
+      </section>
+    </header>
+    <!-- home page -->
+    <section class="home-page">
       <nav>
-        <ul class="menu">
+        <ul class="desktop-navigation">
           <li>Portfolio</li>
           <li>About</li>
           <li>Contact</li>
```

Comment:
Kindly use an element for the Portfolio, About and Contact elements here. They need to go the related area on the page. You can use anchor tag to do that.
- Also please make sure font-size and font-weight matches with the Figma template.
<img width="755" alt="Screenshot 2023-02-26 at 12 02 16 AM" src="https://user-images.githubusercontent.com/25887253/221380136-2cbf1284-ffc6-40f3-ac9e-bd0b9be8e7ba.png">
<img width="1007" alt="Screenshot 2023-02-26 at 12 02 57 AM" src="https://user-images.githubusercontent.com/25887253/221380137-2b1f2b53-9c5d-4d29-81f9-11a7ec2b3052.png">

Reply:
Done